app/screens/home/tab_bar/account.test.tsx

```diff
@@ -0,0 +1,93 @@
+import {renderWithEverything, renderWithIntl} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+import {changeOpacity} from '@utils/theme';
+
+import {Account, default as AccountWithDatabaseObservable} from './account';
+
+import type {UserModel} from '@database/models/server';
+import type Database from '@nozbe/watermelondb/Database';
+
+jest.mock('@components/profile_picture', () => 'ProfilePicture');
+
+const currentUser = {
+    id: 'user1',
+    username: 'testuser',
+} as UserModel;
+
+const theme = {
+    buttonBg: '#000',
+    centerChannelColor: '#fff',
+} as Theme;
+
+describe('Account', () => {
+    const centerChannelColorWithOpacity = changeOpacity(theme.centerChannelColor, 0.48);
+
+    it('should render correctly when focused', () => {
+        const wrapper = renderWithIntl(
+            <Account
+                currentUser={currentUser}
+                isFocused={true}
+                theme={theme}
+            />,
+        );
+
+        const container = wrapper.getByTestId('account-container');
+        expect(container.props.style).toContainEqual({borderColor: theme.buttonBg});
+    });
+
+    it('should render correctly when unfocused', () => {
+        const {getByTestId} = renderWithIntl(
+            <Account
+                currentUser={currentUser}
+                isFocused={false}
+                theme={theme}
+            />,
+        );
+
+        const container = getByTestId('account-container');
+        expect(container.props.style).toContainEqual({borderColor: centerChannelColorWithOpacity});
+    });
+
+    it('should render ProfilePicture with correct props', () => {
+        const {getByTestId} = renderWithIntl(
+            <Account
+                currentUser={currentUser}
+                isFocused={true}
+                theme={theme}
+            />,
+        );
+
+        const profilePicture = getByTestId('account-profile-picture');
+
+        expect(profilePicture.props.author).toEqual({id: 'user1', username: 'testuser'});
+        expect(profilePicture.props.showStatus).toBe(true);
+        expect(profilePicture.props.size).toBe(BOTTOM_TAB_PROFILE_PHOTO_SIZE);
+        expect(profilePicture.props.statusSize).toBe(BOTTOM_TAB_STATUS_SIZE);
+    });
+});
+
+describe('Account with database observable', () => {
+    let database: Database;
+
+    beforeAll(async () => {
+        const server = await TestHelper.setupServerDatabase();
+        database = server.database;
+    });
+
+    it('should render with the correct fake user when observing data from the database', () => {
+        const wrapper = renderWithEverything(
+            <AccountWithDatabaseObservable
+                theme={theme}
+                isFocused={true}
+            />, {database});
+
+        const profilePicture = wrapper.getByTestId('account-profile-picture');
+
+        expect(profilePicture.props.author.email).toContain('@simulator.amazonses.com');
```

Comment:
This is one of the few consistent data from the fakeUser method.  This is just to make sure the observable is working.

Reply:
To make the test more "stable" and a bit more intention revealing, you could add to the database a new user, and set the current user id to that user. That way you don't have to rely on "the few consistent data from the fakeUser method".

Reply:
Make sense too.  Thank you for your feedback.  Although the goal of this is to ensure that the prop `author` receive an object of `UserModel`.  I could check that it is not `undefined` because currentUser can be.

Now that I think about it, I should think about currentUser being `undefined` in Account, however since I'm mocking ProfilePicture, it will not mean a thing.

I can also check that the `author` has a few keys, like email, firstName & lastName.  What the values are irrelevant.